tests/Settings/TestSettingDefinitionsModel.py

```diff
@@ -54,6 +61,8 @@ def test_getAndSet(data):
     # Ensure that the value got set
     assert getattr(model, data["attribute"]) == data["value"]
 
+    # Check properties via function calls
+
```

Comment:
You're missing some tests here.

Reply:
That's some dev code. Removed in 448bc041